restapi.go

```diff
@@ -2814,6 +2814,8 @@ func (s *Session) ApplicationCommands(appID, guildID string) (cmd []*Application
 		endpoint = EndpointApplicationGuildCommands(appID, guildID)
 	}
 
+	endpoint += "?with_localizations=true"
+
 	body, err := s.RequestWithBucketID("GET", endpoint, nil, endpoint)
```

Comment:
I think would be better to leave out the query params out of the bucket name (second `endpoint` specifies the bucket name that will be used when handling the ratelimit).
```suggestion
	body, err := s.RequestWithBucketID("GET", endpoint + "?with_localizations=true", nil, endpoint)
```